tools/devops/automation/build-pipeline.yml

```diff
@@ -220,6 +220,10 @@ extends:
           - repository: yaml-templates
           - repository: sdk-insertions
           - repository: macios-adr # macios-adr has it's own build pipeline and should not be included in the source analysis
+      codeql:
```

Comment:
Shouldn't this be in the build-pr-pipeline.yml?

Reply:
True, was too early, I'll update that